sycl/include/sycl/ext/oneapi/experimental/fixed_size_group.hpp

```diff
@@ -64,7 +64,7 @@ template <size_t PartitionSize, typename ParentGroup> class fixed_size_group {
 
   range_type get_group_range() const {
 #ifdef __SYCL_DEVICE_ONLY__
-    return __spirv_SubgroupMaxSize() / PartitionSize;
+    return __spirv_SubgroupSize() / PartitionSize;
```

Comment:
@Pennycook - Just to double check: This was the intended behavior, right?

Reply:
Good question...  😆 

I had to go and read the spec to figure this out.  It says:
> `group.get_local_linear_range()` must be evenly divisible by `PartitionSize`.

...so, I think you're right.  We don't have to worry about the case where the sub-group size doesn't divide by the partition size, but we _do_ have to worry about the case where the work-group doesn't divide by the sub-group size.